### DISCUSSION TITLE
Uno Library project dependency .xaml/.xbf not updated

### DISCUSSION THREAD

**Discussion by brinko99** [unoplatform/uno#15878] [Q&A] [answered]
I have a Uno app (VS 2022 / .NET 8 / Uno 5.1.87 / Windows head) with a project reference to an Uno Library.

I find that the library's Pages and related content (.xaml / .xbf) are not reliably copied to the output folder of the main app. These files are copied on a full rebuild. However on an incremental build, though changes are detected and the library is built (and it's DLL copied), the .xaml/.xbf files become stale.

My workflow during development requires a full rebuild of the library on every change to the library which appears to allow the main app to detect the change. I've considered post-build steps, but that seems excessive.

Is this expected behavior?

As a repo (attached), I created a blank Uno app and Library.  Upon build, the relevant files are  copied to:
`\UnoApp8.Windows\bin\x64\Debug\net8.0-windows10.0.19041.0\win-x64\UnoLibrary1\Views`

With a change to the Library's Page, upon an incremental build of the App, the above-mentioned files are not updated.

Your help is appreciated!

[UnoApp8.zip](https://github.com/unoplatform/uno/files/14594553/UnoApp8.zip)

**Comment by jeromelaban** [comment, marked as answer]
Thanks for the report. This looks like an issue related to https://github.com/unoplatform/uno/discussions/15807#discussioncomment-8715083. Could you try the workaround and let us know if it helps?

**Reply by brinko99** [reply]
Yep, that fixes it. I added the following to `Directory.Build.props`

```
<PropertyGroup>
   <AccelerateBuildsInVisualStudio>false</AccelerateBuildsInVisualStudio>
</PropertyGroup>
```

Build Acceleration was enabled in VS under "Preview Features"

Thanks as always
